src/render/react/components/Input/comp.js

```diff
@@ -6,7 +6,7 @@ import {
   styleGetterProp,
 } from "../config";
 
-const bridge = globalThis.SJSJSBridge;
+const bridge = globalThis[Symbol.for('lvgljs')];
```

Comment:
kind of similar to being idiomatic. perhaps we want to export these as part of the react runtime package?